tests/integration/api_repo_secrets_test.go

```diff
@@ -73,6 +73,33 @@ func TestAPIRepoSecrets(t *testing.T) {
 		}
 	})
 
+	t.Run("CreateWithDescription", func(t *testing.T) {
+		cases := []struct {
+			Name           string
+			Description    string
+			ExpectedStatus int
+		}{
+			{
+				Name:           "no_description",
+				Description:    "",
+				ExpectedStatus: http.StatusCreated,
+			},
+			{
+				Name:           "description",
+				Description:    "some description",
+				ExpectedStatus: http.StatusCreated,
+			},
```

Comment:
Could you add a test that leaves out the description field, just to ensure that it'll fall back to the default?

Reply:
These tests only check if descriptions can be written. In reality, the default tests written at [api_repo_secrets_test.go#L33](https://github.com/LaoQi/gitea/blob/7771ad9823c36606cae76c2646d0a10e385b4e82/tests/integration/api_repo_secrets_test.go#L33) are cases without descriptions. I haven’t thought of any other test cases yet. Any suggestions?